catalyst/dl/core/experiment.py

```diff
@@ -19,74 +19,115 @@ class Experiment(ABC):
+        """Returns the state parameters for a given stage"""
         pass
 
     @abstractmethod
-    def get_model(self, stage: str) -> _Model:
+    def get_model(self, stage: str) -> Model:
+        """Returns the model for a given stage"""
         pass
 
     @abstractmethod
-    def get_criterion(self, stage: str) -> _Criterion:
+    def get_criterion(self, stage: str) -> Criterion:
+        """Returns the criterion for a given stage"""
         pass
 
     @abstractmethod
-    def get_optimizer(self, stage: str, model: nn.Module) -> _Optimizer:
+    def get_optimizer(self, stage: str, model: Model) -> Optimizer:
+        """Returns the optimizer for a given stage"""
         pass
 
     @abstractmethod
-    def get_scheduler(self, stage: str, optimizer) -> _Scheduler:
+    def get_scheduler(self, stage: str, optimizer: Optimizer) -> Scheduler:
+        """Returns the scheduler for a given stage"""
         pass
 
     def get_experiment_components(
         self, model: nn.Module, stage: str
-    ) -> Tuple[_Criterion, _Optimizer, _Scheduler]:
+    ) -> Tuple[Criterion, Optimizer, Scheduler]:
+        """
+        Returns the tuple containing criterion, optimizer and scheduler by
+        giving model and stage.
+        """
         criterion = self.get_criterion(stage)
         optimizer = self.get_optimizer(stage, model)
         scheduler = self.get_scheduler(stage, optimizer)
         return criterion, optimizer, scheduler
 
     @abstractmethod
     def get_callbacks(self, stage: str) -> "OrderedDict[str, Callback]":
+        """Returns the callbacks for a given stage"""
         pass
 
     def get_datasets(
         self,
         stage: str,
         **kwargs,
     ) -> "OrderedDict[str, Dataset]":
+        r"""Returns the datasets for a given stage and **kwargs"""
         raise NotImplementedError
 
     @abstractmethod
     def get_loaders(self, stage: str) -> "OrderedDict[str, DataLoader]":
+        """Returns the loaders for a given stage"""
         raise NotImplementedError
 
     @staticmethod
     def get_transforms(stage: str = None, mode: str = None):
+        """Returns the data transforms for a given stage and mode"""
         raise NotImplementedError
 
+    def get_native_batch(self, stage: str, loader: Union[str, int] = 0):
+        """Returns a batch from experiment loader
+
+        Args:
+            stage (str): stage name
+            loader (Union[str, int]): loader name or its index,
+                default is the first loader
+        """
+        loaders = self.get_loaders(stage)
+        if isinstance(loader, str):
+            _loader = loaders[loader]
+        elif isinstance(loader, int):
+            _loader = list(loaders.values())[loader]
+        else:
+            raise TypeError("Loader parameter must be a string or an integer")
+
+        dataset = _loader.dataset
+        collate_fn = _loader.collate_fn
+
+        sample = collate_fn([dataset[0]])
```

Comment:
if we add such method, what do you think about 
```
def get_native_batch(self, stage: str, loader: Union[str, int] = 0, batch_index: int):
sample = collate_fn([dataset[batch_index]])
```

by this way we can easily check the data samples from the Experiment

Reply:
ok